build-tools/packages/build-cli/src/library/releaseTag.ts

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * A TSDoc tag representing the release level of an API.
+ */
+export const ReleaseTag = {
```

Comment:
As I was looking at release tags earlier today ( https://github.com/microsoft/FluidFramework/releases/tag/server_v7.0.0 for example) I have released we have two different things we call release tags.

It seems like this PR is introducing the second one? I guess this aligns with API extractor, but personally I'd prefer something else, maybe include "level" in the name?

Maybe one of:

- ApiLevel
- ApiLevelTag
- ReleaseLevelTag
- ReleaseLevel
- SupportLevel (matching https://fluidframework.com/docs/build/releases-and-apitags#api-support-levels )
- ApiSupportLevel
- ApiSupportLevelTag